### PR TITLE
feat: expanded topics and interests

### LINKED ISSUE #927: Feature: Expand interests and topics
### Type of feature

🍕 Feature

### Current behavior

Can we update the interest and topic list to include the top languages on github. 
Screen Shot 2023-02-28 at 7 58 45 PM

### Suggested solution

- [interest](https://github.com/open-sauced/insights/blob/eff13d5922b6453c12aca5a76be38af9db3c96be/lib/utils/getInterestOptions.ts#L2)
- [topics](https://github.com/open-sauced/insights/blob/eff13d5922b6453c12aca5a76be38af9db3c96be/lib/utils/getTopicOptions.ts#L2)

Ruby - 🌹
Java - ☕️
JavaScript - 👾 (updated from ☕️)
CPP (C++) - 🤓
PHP - 📄
CSharp (C#) - 🎹

## Interests
Screen Shot 2023-02-28 at 8 03 19 PM

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR expands the `topic` and `interest` list.

The updated list for both topic and interest:
["javascript", "python", "java", "typescript", "csharp", "cpp", "php", "c", "ruby" , "ai", "ml", "react"]

## Related Tickets & Documents
Fixes #927 #930 

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by bdougie — APPROVED** [review]

**Comment by brandonroberts** [comment]
@shamimbinnur will you reapply the explore fix here?

**Comment by shamimbinnur** [comment]
> @shamimbinnur will you reapply the explore fix here?

@brandonroberts just did it!

File: components/organisms/TopNav/top-nav.tsx [hunk 1]
```diff
@@ -23,11 +23,11 @@ const TopNav: React.FC = () => {
         <div className="flex gap-3 md:gap-8 items-center flex-wrap ">
           <HeaderLogo withBg={false} textIsBlack />
           {!!user && onboarded ? (
-            <>
+            <>``
```
Comment (brandonroberts) [inline components/organisms/TopNav/top-nav.tsx:26]: Revert this change

**Review by brandonroberts — APPROVED** [review]
